app/index.js

```diff
@@ -27,7 +27,7 @@ app.get('/exchange', async function(req, res) {
   const code = req.query.code;
   // in a production app you'll want to store this in some kind of persistent storage
   access = await client.exchangeCode(code)
-  res.sendStatus(200);
+  res.redirect('/vehicle');
```

Comment:
This is a really good usability change lol. Make sure to update the documentation center guide to match this. Also probably worth making this change in the other getting started guides as well

https://smartcar.com/docs/tutorials/node/authorization/#3-receive-the-authorization-code